samples/grids/grid/editing-excel-style-custom/src/index.ts

```diff
@@ -0,0 +1,104 @@
+import 'igniteui-webcomponents-grids/grids/combined';
+import { ComponentRenderer, WebGridDescriptionModule } from 'igniteui-webcomponents-core';
+import { IgcCellType, IgcGridComponent } from 'igniteui-webcomponents-grids/grids';
+import { NwindData } from './NwindData';
+
+import "igniteui-webcomponents-grids/grids/themes/light/bootstrap.css";
+
+import "./index.css";
+
+export class Sample {
+
+    private grid1: IgcGridComponent
+    private _bind: () => void;
+
+    constructor() {
+        var grid1 = this.grid1 = document.getElementById('grid1') as IgcGridComponent;
+    
+        this._bind = () => {
+            grid1.data = this.nwindData;
+
+            grid1.addEventListener('activeNodeChange', (event: any) => {
+                grid1.endEdit();
+                grid1.markForCheck();
```

Comment:
Currently after exiting edit mode here, focus is lost and navigation cannot continue. You can move the focus on the tbody of the grid, for example:

```
(grid1.getElementsByClassName("igx-grid__tbody-content")[0] as any).focus();
```
So that navigation can continue.

Reply:
Updated as per your suggestion